front/public/locales/de/operationalStudies/manageTrainSchedule.json

```
@@ -0,0 +1,170 @@
+  "infraLoading": "Betriebsnetz wird geladen…",
+  "inputOPTrigrams": "Geben Sie die Dreierkürzel der zu durchfahrenen Betriebsstellen ein, getrennt nach Leerzeichen.",
+  "inputOPTrigramsExample": "Bsp: HHA HGÖ FKS",
+  "incompatibleConstraints": {
+    "filters": {
+      "activateAll": "Alle Beschränkungen anzeigen",
+      "disable": "Beschränkungen des Typs '{{name}}' ausblenden",
+      "enable": "Beschränkungen des Typs '{{name}}' anzeigen",
+      "focus": "Nur Beschränkungen des Typs '{{name}}' anzeigen",
+      "title": "Nach Beschränkungsart filtern"
+    },
+    "incompatible_electrification_ranges": "Bahnstromart nicht möglich",
+    "incompatible_gauge_ranges": "Fahrzeugbegrenzungslinie",
+    "incompatible_signaling_system_ranges": "Signalsystem nicht unterstützt",
+    "info": "Wegefindung aufgrund gegebener Beschränkungen fehlgeschlagen. Sie finden hier eine Liste der Beschränkungen, die beim kürzesten Weg fehlgeschlagen sind; für jede wird hierbei die Art, die Start- und Endposition des Fehlergebiets in Bezug zum Wegestart und die Länge angegeben.",
+    "gotoMap": "Auf Karte ansehen",
+    "noElectrification": "Kein Bahnstrom",
+    "seeConstraintsOnMap": "Beschränkungen auf Karte ansehen",
+    "title": "{{total}} verletzte Beschränkungen",
+    "title_on": "{{count}} von {{total}} verletzter Beschränkungen"
+  },
+  "inverseOD": "Route umkehren",
+  "launchPathFinding": "Wegefindung starten",
+  "launchSimulation": "Start",
+  "manageVias": "Wegepunkteverwaltung",
+  "missingPathSteps": "Mindestens ein Wegepunkt wurde im Netz nicht gefunden.",
+  "noPlaceChosen": "Kein Wegepunkt/Endhalt definiert",
+  "noOriginChosen": "Herkunftsort nicht definiert",
+  "noDestinationChosen": "Ziel nicht definiert",
+  "noPowerRestriction": "Keine Kraftbegrenzung",
+  "noSpeedLimitByTag": "Kein",
+  "noTimetable": "Fahrplan auswählen",
+  "noTrainCompo": "Keine Zugzusammensetzung definiert…",
+  "origin": "Herkunftsort",
+  "pacedTrains": {
+    "cadence": "Bedienungstakt",
+    "defineService": "Bedienung erstellen",
+    "timeRangeDuration": "Taktlänge"
+  },
+  "pathfinding": "Wegefindung",
+  "pathfindingDone": "Wegefindung durchgeführt.",
+  "pathfindingError": "Ein Fehler trat bei der Wegefindung auf: {{errorMessage}}.",
+  "InvalidTrainScheduleStep": "Mindestens ein Wegepunkt konnte nicht erkannt werden.",
+  "pathfindingInProgress": "Wegefindung läuft…",
+  "pathfindingMissingParams": "Fehlende Parameter für Wegefindung: {{missingElements}}.",
+  "pathfindingMissingParamsSimple": "Fehlende Parameter für Wegefindung",
+  "pathfindingNoState": "Keine Wegefindung.",
+  "pathfindingStatus": "Status der Wegefindung",
+  "pathfindingErrors": {
+    "incompatible_constraints": "In Konflikt mit Beschränkungen",
+    "invalid_path_item": "Wegepunkt nicht gefunden.",
+    "not_found_in_blocks": "Fehlender Streckenblock.",
+    "not_found_in_routes": "Fehlende Route.",
+    "not_found_in_tracks": "Fehlendes Gleis.",
```

Same as station-specific

```suggestion
    "not_found_in_tracks": "Fehlende Strecke.",
```